input/rmatch.in

```diff
@@ -1,94 +1,94 @@
+    match_sol%connect_threshold=1e-3 ! Proportional difference between outer and inner solutions for match_auto_connect to connect them
+    match_sol%b_flag=f   ! Write solutions in terms of b instead of xi
+    coil%ideal_flag=f    ! Write ideal galerkin solution instead of resistive -- combined with GPEC this is physically equivalent to running regular DCON
 /
 &RMATCH_OUTPUT
-    bin_rpecsol=f
-    out_rpecsol=f
+    bin_rpecsol=f        ! Write outer and inner solutions to binary files
+    out_rpecsol=f        ! Write outer and inner solutions to ascii files
 /
 &DELTAC_LIST
-    tid=3,5,6,4
-    deltac_bin_sol=f
-    deltac_out_sol=f
+    tid=3,5,6,4          ! Something to do with which solution indexes are used in computing combined solutions. Always 3,5,6,4 in examples.
+    deltac_bin_sol=f     ! Write deltac solutions to binary files for xdraw
+    deltac_out_sol=f     ! Write deltac solutions to ascii files
 
-    rescale=t
-    pfac=0.1
-    xfac=10.0
-    gal_method="normal"
-    noexp=t
-    nx=128
-    nq=5
-    deltac_tol=1e-11
-    dx1dx2_flag=f
-    dx1=0.001
-    dx2=0.002
-    restore_ul=t
-    restore_uh=t
-    restore_us=t
+    rescale=t            ! Rescale x during asymptotic matching for better numerical behavior
+    pfac=0.1             ! Packing ratio? Should it be the same as in RDCON?
+    gal_method="normal"  ! Something to do with cell types used? Options are "normal", "resonant". "normal" in examples.
+    noexp=t              ! Unknown function. Always true in examples.
+    nx=128               ! The number of elements in each interval between two singular surfaces
+    nq=5                 ! The number of Gaussian points in each Galerkin element
+    deltac_tol=1e-11     ! Tolerance of lsode integration
+    dx1dx2_flag=f        ! Flag to include the special dx1 and dx2 treatments for resonant and extension element
+    dx1=0.001            ! The size of resonant element
+    dx2=0.002            ! The size of extension element
+    restore_ul=t         ! Include the Hermite solution in Galerkin solution
+    restore_uh=t         ! Include the small solution in Galerkin solution
+    restore_us=t         ! Include the larger solution in Galerkin solution
 
-    cutoff=20
+    cutoff=20            ! The number of elements include the large solution as the driving term
 
-    interp_np=5
-    order_pow=20
-    order_exp=20
+    interp_np=5          ! The number of interpretation points for outputting Galerkin solution
+    order_pow=20         ! The order of polynomial expansion of large solution in asymptotic matching
+    order_exp=20         ! The order of exponential expansion of small solution in asymptotic matching
 
-    diagnose_res=f
+    diagnose_res=f       ! Diagnose asymptotic solutions in ua.out/ua.bin
 
-    x0_ua=0.3
-    x1_ua=1
-    nx_ua=100
-    outt=3
+    x0_ua=0.3            ! Unknown function for diagnose_res
+    x1_ua=1              ! Unknown function for diagnose_res
```

Comment:
Furthest distance from rational surface where the asymptotic approximation is evaluated in function inpso_ua_diagnose